src/animation/friction.ts

```diff
@@ -41,18 +39,17 @@ const applyFrictionForceForStep = (
  */
 export const friction1D = (
   params: Friction1DParams
-): { controller: Controller } => {
+): { controller: AnimationInitializer } => {
   const state: FrictionState = {
     mover: {
       mass: params.config.mass,
       acceleration: [0, 0],
-      velocity: [params.config.initialVelocity, 0],
+      velocity: [params.config.initialVelocity * 1000, 0],
```

Comment:
In order to get realistic motion previously, our velocity values had to be _super high_. This is because we're calculating the force every millisecond, but our equation assumes seconds as the time unit. Multiplying by 1000 here helps to convert everything to the proper reference units for deriving force.

Reply:
I can’t quite see where the other part of the velocity code is but if you’re using milliseconds it’s probably worth converting to seconds there in the formula? Or are milliseconds already established as the base time frame there? Kind of unexpected imho judging from a metric standpoint 😅

Reply:
Here's the other place where it's used: https://github.com/FormidableLabs/renature/pull/17/files#diff-3dcc94f37ffa7fc7c99d5f790af6589fR45 Basically, we use the third kinematic equation to answer the question:

> If an object is moving at v_0 m/s on a rough surface, how far will it travel by the time it has reached velocity of 0?

For that formula, the base time frame we're using is `ms`. But you're right, a better solution would be to do the conversion there and establish the base time frame as seconds. The challenge too is that we're applying the force once every `ms` (roughly, assuming no dropped frames). So just some work for me to do to reconcile the units a bit more cleanly. Good shout.